modules/sndfile/sndfile.c

```diff
@@ -168,6 +184,13 @@ static struct aufilt sndfile = {
 static int module_init(void)
 {
 	aufilt_register(&sndfile);
+
+	if (conf_get(conf_cur(), "snd_path", &file_path)) {
```

Comment:
as a general rule in baresip/libre, when using the `struct pl` type you should use the value
in the same function, because the data might be gone when the function returns.

please use a string instead and the function `conf_get_str` to read the config. The return value
can be ignored.